server/models/submission.py

```diff
@@ -234,3 +238,111 @@ def updateFolderAccess(self, phase, submissions):
                     folderModel.save(folder, validate=False)
         except TypeError:
             raise ValidationException('A list of submissions is required.')
+
+    def scoreSubmission(self, submission, apiUrl):
+        """
+        Run a Girder Worker job to score a submission.
+        """
+        folderModel = self.model('folder')
+        jobModel = self.model('job', 'jobs')
+        phaseModel = self.model('phase', 'covalic')
+        settingModel = self.model('setting')
+        tokenModel = self.model('token')
+        userModel = self.model('user')
```

Comment:
We can import these models statically now:

```
from girder.models.folder import Folder

...

Folder().load(...)
```

etc.

Reply:
Thanks for the feedback. Yes, but I maintained the convention in this file/project. Update to modernize for newer Girder versions could be done together and separately.

Reply:
Fair enough. LGTM